newrelic.yml

```diff
@@ -169,7 +169,13 @@ common: &default_settings
   # If true, disables Action Cable instrumentation.
   # disable_action_cable_instrumentation: false
 
-  # If true, disables instrumentation for Active Record 4, 5, and 6.
+  # If true, disables Action Mailbox instrumentation.
+  # disable_action_mailbox: false
+
+  # If true, disables Action Mailer instrumentation.
+  # disable_action_mailer: false
+
+  # If true, disables instrumentation for Active Record 4+
```

Comment:
Since this reference is being updated, could you update the [default_source.rb reference](https://github.com/newrelic/newrelic-ruby-agent/pull/1740/files#diff-d116d3c8c7bebe204a18ee3983ac59c5d15bd70da7123f2a2c4a12f144bca181R1192) as well?